h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -1537,7 +1542,7 @@ private Chunk chunkUUID() {
       if(idx >= 0)i = idx;
       else cancel_sparse(); // for now don't bother setting the sparse value
     }
-    _is[i] = _sslen;
+    if (_is != null) _is[i] = _sslen;
```

Comment:
Reverted this line. But Tomas, what do you think about the next line? Should it still check for null, or better throw another NPE if _is is null? It's a little bit confusing to me, how come in one line we allow npe, and in the next one we check for null.